### ISSUE TITLE
apt gpg key published by mysql is expiring in 90 days (2022-02-16)

### ISSUE THREAD

**Issue by rfay** [docker-library/mysql#801]
The apt gpg key [published and used by mysql](https://dev.mysql.com/doc/refman/8.0/en/checking-gpg-signature.html) is expiring on 2022-02-16 (90 days)

This affects all the mysql images, but also the mysql packages in general. I know that it needs to be fixed by Oracle, but don't know how to report to them. The key in the docker images will need to be updated as well.

`docker run -it --rm  --entrypoint=bash mysql:5.6 apt-key list` will show the keys in the 5.6 image. 

There you see the mysql gpg key expiring 2022-02-16
```
/etc/apt/trusted.gpg.d/mysql.gpg
--------------------------------
pub   dsa1024 2003-02-03 [SCA] [expires: 2022-02-16]
      A4A9 4068 76FC BD3C 4567  70C8 8C71 8D3B 5072 E1F5
uid           [ unknown] MySQL Release Engineering <mysql-build@oss.oracle.com>
```

Please get the powers that be to update this key soon!

If you need to direct me to another reporting technique, do it :)

**Comment by tianon** [comment]
(Honestly, https://bugs.mysql.com/ is probably the "right" place -- that should get the attention of the infra folks who can do something about it. :sweat_smile:)

**Comment by rfay** [comment]
Created https://bugs.mysql.com/bug.php?id=105632 - thanks!

This of course will mean new work for the official mysql images as well, so might as well keep this open.

Impressive that they let the keys expire the last time around...

**Comment by ltangvald** [comment]
Yeah, "impressive" was definitely how we felt about it, too *cough*. We're a bit more on top of this now, though, so the package and key should be updated for the January release

**Comment by tianon** [comment]
Nice, I'll go ahead and close this then; thanks @ltangvald :heart:

**Comment by rfay** [comment]
Noting that when the key is published maintenance will be required on these images...

**Comment by rfay** [comment]
I guess it looks like they're just going to let it expire? I wonder if anybody has any contacts that can push them a bit.

**Comment by ltangvald** [comment]
Updates to the apt repo follow the regular MySQL release cycle, so a new gpg key will be published together with the 8.0.28 release. One important note though is that I believe this will actually be a _new_ key, so we will need a change for it, but I'll try to prepare a PR for this.

**Comment by ltangvald** [comment]
Added PR #808
